src/components/rangeFilterInput/__tests__/Index.Test.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import { render, screen, waitFor } from '@testing-library/react';
+
+import RangeFilterInput from '../index';
+
+describe('<RangeFilterInput/>', () => {
+  const mockOnChange = jest.fn();
+
+  const renderInputField = () => {
+    return render(
+      <RangeFilterInput
+        handleChange={mockOnChange}
+        name={{
+          from: 'priceFrom',
+          to: 'priceTo',
+        }}
+        unit="CHF"
+        value={{
+          from: 200,
+          to: 1000,
+        }}
+        placeholder={{
+          from: 'From',
+          to: 'To',
+        }}
+      />
+    );
+  };
+
+  beforeEach(() => {
+    mockOnChange.mockClear();
```

Comment:
You could pass the `mockOnChange` as an argument to `renderInputField` (with a default `jest.fn()`). This way each test that relies on it could  create its own local mock and there wouldn't be a need to clear the mock between tests

Reply:
This comment has not been fully addressed yet.
Instead of creating a global mock in line 10, please create mocks inside the individual tests that need them